soci/soci_index.go

```diff
@@ -51,32 +49,60 @@ const (
 	IndexAnnotationBuildToolIdentifier = "com.amazon.soci.build-tool-identifier"
 	// index annotation for build tool version
 	IndexAnnotationBuildToolVersion = "com.amazon.soci.build-tool-version"
+	// media type for OCI Reference Types manifest
+	OCIRefTypeManifestMediaType = "application/vnd.oci.artifact.manifest.v1+json"
```

Comment:
We here in 2022 know what "OCIRefType" and "OCI Reference Types" means, because we remember the name of the working group that produced it. But the future versions of ourselves will be confused, and will have trouble googling that phrase to get unconfused. Let's use the term that [the spec](https://github.com/opencontainers/image-spec/blob/ff0257194e20f7fd17ca21c67a2e725f21c4d4f2/artifact.md#oci-artifact-manifest-specification) uses: `OCI Artifact`

Reply:
Done.